python-bindings/pythonbindings.cpp

```diff
@@ -55,17 +45,19 @@ PYBIND11_MODULE(blspy, m)
                     throw std::invalid_argument(
                         "Length of bytes object not equal to PrivateKey::SIZE");
                 }
+                py::gil_scoped_release release;
                 auto data_ptr = reinterpret_cast<const uint8_t *>(info.ptr);
```

Comment:
info is still in python right?

Reply:
Hmm i assumed that the `PyObject_GetBuffer` in `buffer::request` copies it over but it might not from a quick deeper look at the code.. will review again tomorrow!

Reply:
Okay changed it, see 353a623. There were even some more cases were `info.ptr` was passed into `Bytes` and then into `::FromBytes` which still would have been an access to that memory without the GIL (`Bytes` only stores the pointer).